shpool/tests/attach.rs

```diff
@@ -554,7 +554,8 @@ fn default_keybinding_detach() -> anyhow::Result<()> {
         lm1.scan_until_re("someval$")?;
 
         a1.run_raw_cmd(vec![0, 17])?; // Ctrl-Space Ctrl-q
-        a1.proc.wait()?;
+        let exit_status = a1.proc.wait()?;
+        assert!(exit_status.success());
```

Comment:
just curious, does this run at runtime for users? (I assume this is like python assert which only runs if debug mode is set i think)

Reply:
No it is never compiled out (there is [debug_assert!](https://doc.rust-lang.org/std/macro.debug_assert.html) if you want that functionality). Here it is just in a test, so not part the main application though, so doesn't really matter either way.

Reply:
ah yes that makes sense, sg!